medmodels/treatment_effect_estimation/continuous_estimators.py

```diff
@@ -0,0 +1,208 @@
+    to 1 z-score.
+
+    A rule of thumb for interpreting Cohen's D
+    - Small effect = +-0.2
+    - Medium effect = +-0.5
+    - Large effect = +-0.8
+
+    If the difference is negative, indicated the mean in the treated group is lower than
+    the control group.
+
+    This metric provides a dimensionless measure of effect size, facilitating the
+    comparison across different studies and contexts.
+
+    Args:
+        medrecord (MedRecord): An instance of the MedRecord class containing medical
+            data.
+        treatment_true_set (Set[NodeIndex]): A set of node indices representing the
+            treated group.
+        control_true_set (Set[NodeIndex]): A set of node indices representing the
+            control group.
+        outcome_group (Group): The group of nodes that contain the outcome variable.
+        outcome_variable (MedRecordAttribute): The attribute in the edge that contains
+            the outcome variable.
+        reference (Literal["first", "last"], optional): The reference point for the
+            exposure time. Options include "first" and "last". If "first", the function
+            returns the earliest exposure edge. If "last", the function returns the
+            latest exposure edge. Defaults to "last".
+        time_attribute (MedRecordAttribute, optional): The attribute in the edge that
+            contains the time information. Defaults to "time".
+        add_correction (bool, optional): Whether to apply a correction factor for small
+            sample sizes. Defaults to False.
+
+    Returns:
+        float: The Cohen's D coefficient, representing the effect size.
+    """
+    treated_outcomes = np.array(
+        [
+            medrecord.edge[
+                find_reference_edge(
+                    medrecord,
+                    node_index,
+                    outcome_group,
+                    time_attribute=time_attribute,
+                    reference=reference,
+                )
+            ][outcome_variable]
+            for node_index in treatment_true_set
+        ]
+    )
+    if not all(isinstance(i, (int, float)) for i in treated_outcomes):
+        raise ValueError("Outcome variable must be numeric")
+
+    control_outcomes = np.array(
+        [
+            medrecord.edge[
+                find_reference_edge(
+                    medrecord,
+                    node_index,
+                    outcome_group,
+                    time_attribute="time",
+                    reference=reference,
+                )
+            ][outcome_variable]
+            for node_index in control_true_set
+        ]
+    )
+    if not all(isinstance(i, (int, float)) for i in treated_outcomes):
+        raise ValueError("Outcome variable must be numeric")
+
+    min_len = min(len(treated_outcomes), len(control_outcomes))
+    cf = 1  # correction factor
+
+    if min_len < 50:
+        logging.warning(
```

Comment:
We're not doing logging yet. We first need to research more and implement a proper logging setup. Can you remove this and replace it with a TODO and the current log?

Reply:
(and remove all the other logging stuff) :)